lib/twitter_cldr/js/mustache/bundle.coffee

```diff
@@ -30,6 +30,16 @@ TwitterCldr.locale = "{{current_locale}}";
 {{> utilities}}
 {{{contents}}}
 
+TwitterCldr.set_data = (bundle) ->
+  TwitterCldr.data = bundle
+  null
+
+TwitterCldr.get_data = ->
+  if (TwitterCldr.data)?
```

Comment:
I don't think you need parentheses here. Also, the next few lines seem to have 4 spaces indentation.